nf_core/components/components_command.py

```diff
@@ -48,24 +59,25 @@ def _configure_repo_and_paths(self, nf_dir_req=True):
                 raise
             self.repo_type = None
             self.org = ""
-        self.default_modules_path = Path("modules", self.org)
-        self.default_tests_path = Path("tests", "modules", self.org)
-        self.default_subworkflows_path = Path("subworkflows", self.org)
-        self.default_subworkflows_tests_path = Path("tests", "subworkflows", self.org)
+        self.default_modules_path: Path = Path("modules", self.org)
+        self.default_tests_path: Path = Path("tests", "modules", self.org)
+        self.default_subworkflows_path: Path = Path("subworkflows", self.org)
+        self.default_subworkflows_tests_path: Path = Path("tests", "subworkflows", self.org)
 
-    def get_local_components(self):
+    def get_local_components(self) -> List[str]:
         """
         Get the local modules/subworkflows in a pipeline
         """
-        local_component_dir = Path(self.dir, self.component_type, "local")
+        local_component_dir: Path = Path(self.dir, self.component_type, "local")
         return [
             str(path.relative_to(local_component_dir)) for path in local_component_dir.iterdir() if path.suffix == ".nf"
         ]
 
-    def get_components_clone_modules(self):
+    def get_components_clone_modules(self) -> List[str]:
         """
         Get the modules/subworkflows repository available in a clone of nf-core/modules
         """
+        component_base_path: Path
```

Comment:
Is this correct/necessary? The variable exists only within the scope of this method and technically I think a `str` wouldn't break anything even though we prefer/expect a `Path`?